packages/astro/src/core/build/generate.ts

```diff
@@ -244,7 +246,13 @@ async function generatePage(
 			const timeEnd = performance.now();
 			const timeChange = getTimeStat(prevTimeEnd, timeEnd);
 			const timeIncrease = `(+${timeChange})`;
-			logger.info('SKIP_FORMAT', ` ${dim(timeIncrease)}`);
+			let timeIncreaseLabel;
+			if (Math.round(timeEnd - prevTimeEnd) > THRESHOLD_SLOW_RENDER_TIME_MS) {
```

Comment:
Do we need to call `Math.round` when comparing numbers? I think it should also work without